### DISCUSSION TITLE
非北京的阿里云服务器有可以用的内部上传方式吗

### DISCUSSION THREAD

**Discussion by crackzhazha** [AlistGo/alist#4239] [Q&A]
根据源码可以看到在北京内的vps可以使用内部oss地址调用阿里云盘api，但是这个只能在北京区域的vps访问，有没有其他区域的链接呢？
https://github.com/alist-org/alist/blob/36cd504783962818180a648df6fe9cea57c203d9/drivers/aliyundrive_open/util.go#L139
uploadUrl = strings.ReplaceAll(uploadUrl, "https://cn-beijing-data.aliyundrive.net/", "http://ccp-bj29-bj-1592982087.oss-cn-beijing-internal.aliyuncs.com/")

**Comment by anwen-anyi** [comment]
没有 因为阿里云盘用的就是北京对象存储 
所以只能使用北京地区的 
其他地区没办法

**Reply by crackzhazha** [reply]
那我看现在只有upload使用了内部上传，可不可以把其他操作也支持内部的访问形式呢？我是把阿里云盘挂在我的vps上，当一个低端的NAS在用，放一些低频访问的大文件
